apiary.apib

```diff
@@ -1567,6 +1567,30 @@ Get a list of all notes.
                     + id: `36386b05-936e-4cc0-9523-bd20d797ebf5` (string)
                 + added_at: `2016-01-01T00:00:00+00:00` (string)
 
+### notes.add [POST /notes.add]
```

Comment:
Following the reasoning from create vs add, this will have to be `create`

Create => doesn't exist out of TLF
Add => does exist out of TLF

We could argue a note can exist out of TLF but I think create is the more applicable here.